libpod/define/container_inspect.go

```diff
@@ -622,6 +622,7 @@ type InspectContainerData struct {
 	OCIRuntime      string                      `json:"OCIRuntime,omitempty"`
 	LogPath         string                      `json:"LogPath"`
 	LogTag          string                      `json:"LogTag"`
+	LogSize         string                      `json:"LogSize"`
```

Comment:
This should actually go in `InspectLogConfig` - https://github.com/containers/podman/blob/15ac7c5313d58c12f46dfa7eaa6ade05aca3df59/libpod/define/container_inspect.go#L84-L89

Reply:
I don't know if it should be part of Config or a separate field, though. Either works for me.

Reply:
You want it in a different place the LogPath and LogTag?

Reply:
Yes. LogConfig is the more correct place (it's where `docker inspect` sticks all their log settings). We should probably move the other options in there as well.

Reply:
@rhatdan, I thought @mheon wanted it moved here?  https://github.com/containers/podman/blob/5cf7aa65fb4786ee3dad191a725f6c37fae83bc3/pkg/specgen/specgen.go#L14
Or am I mistaken?

Reply:
Yes you are correct. I missed that.